tests/meta/demultiplex/test_housekeeper_storage_functions.py

```diff
@@ -213,3 +216,30 @@ def test_add_demux_logs_to_housekeeper(
     assert len(files) == 2
     for file in files:
         assert file.path.split("/")[-1] in expected_file_names
+
+
+def test_store_fastq_path_in_housekeeper_correct_tags(
+    populated_housekeeper_api, empty_fastq_file_path, novaseq6000_flow_cell, store
+):
+    """Test that a fastq file is stored in Housekeeper with the correct tags."""
+    sample_id: str = "sample_internal_id"
+    # GIVEN a fastq file that has not been added to Housekeeper
+    assert not populated_housekeeper_api.files(path=empty_fastq_file_path.as_posix()).first()
+
+    # WHEN adding the fastq file to housekeeper
+    with mock.patch(
+        "cg.meta.demultiplex.housekeeper_storage_functions.check_if_fastq_path_should_be_stored_in_housekeeper",
+        return_value=True,
+    ):
+        store_fastq_path_in_housekeeper(
+            sample_internal_id=sample_id,
+            sample_fastq_path=empty_fastq_file_path,
+            flow_cell=novaseq6000_flow_cell,
+            hk_api=populated_housekeeper_api,
+            store=store,
+        )
+
+    # THEN the file was added to Housekeeper with the correct tags
+    file: File = populated_housekeeper_api.get_files(bundle=sample_id).first()
+    expected_tags: Set[str] = {SequencingFileTag.FASTQ.value, novaseq6000_flow_cell.id, sample_id}
+    assert set([tag.name for tag in file.tags]) == expected_tags
```

Comment:
Very minor but would ensure that we do not miss duplicated tags.
```suggestion
    assert [tag.name for tag in file.tags] == list(expected_tags)
```

Reply:
Good point. I am not sure if we can duplicate tags but I chose set is because if the tags are not in the right order the list comparison will fail

Reply:
Oh right, that is probably better then!